packages/components/src/InputFile/InputFileButton.tsx

```diff
@@ -21,7 +21,6 @@ export function InputFileButton({
   return (
     <Button
       {...buttonProps}
-      ariaLabel="Upload file"
       label={label || contextLabel}
```

Comment:
Remove aria-label here because of https://accessibleweb.com/question-answer/do-i-need-aria-attributes-or-roles-on-native-elements/

This change was breaking tests and buttons by default use the text content as the label so it wasn't needed

Reply:
@jdeichert this is the other commit I made